configure.ac

```
@@ -273,9 +273,9 @@ AC_ARG_ENABLE([unix-lib],
   [AS_HELP_STRING([--disable-unix-lib],
     [do not build the unix library])])
 
-AC_ARG_ENABLE([bigarray-lib],
-  [AS_HELP_STRING([--disable-bigarray-lib],
-    [do not build the legacy separate bigarray library])])
+AC_ARG_ENABLE([bigarray-lib], [],
+  [AC_MSG_ERROR([The bigarray-lib option was deleted in OCaml 5.00.])],
```

```suggestion
  [AC_MSG_ERROR([The bigarray-lib option was deleted in OCaml 5.00, as it is now part of the standard library])],
```

(to make OS porters live's easier)

---

Thanks, reworded to clarify the sentence.